packages/fast-css-editor-react/src/spacing/spacing.props.ts

```diff
@@ -0,0 +1,58 @@
+import { ManagedClasses } from "@microsoft/fast-jss-manager";
+import { CSSSpacingClassNameContract } from "./spacing.style";
+
+export enum SpacingType {
+    margin = "margin",
+    padding = "padding",
+}
+
+export enum SpacingKey {
+    marginTop = "marginTop",
+    marginLeft = "marginLeft",
+    marginRight = "marginRight",
+    marginBottom = "marginBottom",
+    paddingTop = "paddingTop",
+    paddingBottom = "paddingBottom",
+    paddingLeft = "paddingLeft",
+    paddingRight = "paddingRight",
+}
+
+export interface CSSSpacingState {
+    activeType: SpacingType;
+
+    hoverType: SpacingType | undefined;
+}
+
+export interface CSSSpacingUnhandledProps extends React.HTMLAttributes<HTMLDivElement> {}
+
+export interface CSSSpacingValues {
+    marginTop?: string;
```

Comment:
I would probably alphabetize interface properties

Reply:
I'll go with @nalogiudice suggestion for this, it makes sense for CSS sequence to be followed.